docker-compose.yaml

```diff
@@ -475,7 +475,7 @@ services:
     image: ${IMAGE_REPO:-lagoon}/logs-db
     user: '111111111'
     environment:
-      KEYCLOAK_URL: http://docker.for.mac.localhost:8088
+      KEYCLOAK_URL: http://keycloak:8080
```

Comment:
can you explain that change?

Reply:
it comes from #1526 .
GNU/Linux doesn't know `docker.for.mac.localhost` and both `logs-db` and `logs-db-ui` dont' start properly.
To test lagoon on my local, I had to "merge" the changes @smlx made in the PR mentioned.

Reply:
okay, but let's remove it from this PR please, we want to keep them separate

Reply:
Removed